app/router.go

```diff
@@ -41,10 +43,13 @@ func URLMatcher(pattern string) mux.MatcherFunc {
 	}
 }
 
-// Router gives of the HTTP dispatcher. It will always use the embedded HTML
-// resources.
-func Router(c Reporter) *mux.Router {
+// Router returns the HTTP dispatcher, managing API and UI requests, and
+// accepting reports from probes.. It will always use the embedded HTML
+// resources for the UI.
+func Router(c collector) *mux.Router {
 	router := mux.NewRouter()
+	post := router.Methods("POST").Subrouter()
+	post.HandleFunc("/", handleReport(c))
```

Comment:
What do you think about putting this at a different path to /?  How about /api/report?